server/src/main/java/au/org/ala/names/ws/resources/NameSearchResource.java

```diff
@@ -439,6 +443,15 @@ private NameUsageMatch search(NameSearch search) throws Exception {
             result = metrics == null ? null : metrics.getResult();
         }
 
+        // See if the scientific name is actually a LSID
+        if (this.allowLoose && search.isLoose()) {
```

Comment:
I can see how`search.loose` is set up in the tests below, however, how/where `allowLoose` is configured?

Reply:
See README.md line 243